Makefile

```diff
@@ -56,6 +56,8 @@ test: .pdm
 testcov: test
 	@echo "building coverage html"
 	@pdm run coverage html
+	@echo "building coverage lcov"
+	@pdm run coverage lcov
```

Comment:
This one is a suggestion. I would like to add `coverage.lcov` generation. There are tools for IDEs that usually read this format. I use this one https://marketplace.visualstudio.com/items?itemName=markis.code-coverage

Reply:
👍 fine by me